docker/pravega/Dockerfile

```diff
@@ -7,10 +7,13 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-FROM openjdk:8-jre
+FROM openjdk:8-jre-alpine
 
-RUN apt-get update && apt-get install -y -q rpcbind nfs-common \
-    && rm -rf /var/lib/apt/lists/*
+RUN apk add --update \
+    rpcbind \
+    nfs-utils \
+    python \
```

Comment:
I see that `python` is added here whereas it was not present in the earlier Dockerfile. Please validate whether we need this.

Reply:
@adrianmo sorry to ping an old conversation, but wasn't the real question, "does Pravega require Python"?

Reply:
@EronWright Python was added because there's a Python script that needs it. However, I've created https://github.com/pravega/pravega/issues/3251 to replace that Python script with a Bash alternative and eliminate the Python dependency.